go/pkg/uint128/uint128.go

```diff
@@ -0,0 +1,71 @@
+package uint128
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+
+	"github.com/google/uuid"
+)
+
+// Uint128 stores a 128-bit unsigned integer as two 64-bit words. Two Uint128 values can be compared
+// with ==.
+type Uint128 struct {
```

Comment:
One thing I love about Rust is the `u128` built-in type  ;)

Reply:
Maybe it will come to Go eventually: https://github.com/golang/go/issues/9455...